pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -52,12 +51,16 @@
 , libmbim
 , libcbor
 , xz
+
+, breakpointHook
+, ripgrep
 }:
 
 let
   python = python3.withPackages (p: with p; [
     pygobject3
     setuptools
+    markdown
```

Comment:
This is not actually used – it is only attempting to check that `gi-docgen` is using sufficient version of markdown but that is not really possible for us, since `gi-docgen` is self-contained and there is no guarantee that the `markdown` in the environment.

I have opened https://github.com/fwupd/fwupd/pull/5263 but that still will not work without https://github.com/NixOS/nixpkgs/pull/200775.

Reply:
For now, we might something like this https://github.com/NixOS/nixpkgs/pull/192766/files#r979250631, or add a comment that the `markdown` is temporary until https://github.com/fwupd/fwupd/pull/5263 is in.